flowzone.yml

```diff
@@ -53,6 +53,41 @@
       submodules: "recursive"
       token: ${{ secrets.FLOWZONE_TOKEN }}
 
+  - &ifInternalPullRequest
+    if: github.event.pull_request.head.repo.full_name == github.repository
+
+  - &ifPrivateRepository
+    if: github.event.repository.private
+
+  - &loginWithDockerHub
+    name: Login to Docker Hub
+    continue-on-error: true
+    uses: docker/login-action@f4ef78c080cd8ba55a85445d5b36e214a81df20a # v2
+    with:
+      username: ${{ secrets.DOCKERHUB_USER || secrets.DOCKER_REGISTRY_USER }}
+      password: ${{ secrets.DOCKERHUB_TOKEN || secrets.DOCKER_REGISTRY_PASS }}
+
+  - &loginWithGitHubContainerRegistry
+    name: Login to GitHub Container Registry
+    continue-on-error: true
+    uses: docker/login-action@f4ef78c080cd8ba55a85445d5b36e214a81df20a # v2
+    with:
+      registry: ghcr.io
+      username: ${{ github.actor }}
```

Comment:
This changes a bunch of entries from `env.GHCR_USER` to `github.actor` - is that ok?

Reply:
ugh, no